src/test/regress/expected/upgrade_columnar_before.out

```diff
@@ -0,0 +1,143 @@
+INSERT INTO test_retains_data VALUES
+    (1, 'abc', (1, '4'), ARRAY[1,2,3,4]),
+    (2, 'pi', (3, '192'), ARRAY[3,1,4,1,5]),
+    (3, 'earth', (4, '22'), ARRAY[1,2,7,5,6]);
+--
+-- Verify that after upgrade we can read data for tables whose
+-- relfilenode has changed before upgrade.
+--
+-- truncate
+CREATE TABLE test_truncated (a int) USING columnar;
+INSERT INTO test_truncated SELECT * FROM generate_series(1, 10);
+SELECT count(*) FROM test_truncated;
+ count
+---------------------------------------------------------------------
+    10
+(1 row)
+
+SELECT relfilenode AS relfilenode_pre_truncate
+FROM pg_class WHERE oid = 'test_truncated'::regclass::oid \gset
+TRUNCATE test_truncated;
+SELECT relfilenode AS relfilenode_post_truncate
+FROM pg_class WHERE oid = 'test_truncated'::regclass::oid \gset
+SELECT :relfilenode_post_truncate <> :relfilenode_pre_truncate AS relfilenode_changed;
+ relfilenode_changed
+---------------------------------------------------------------------
+ t
+(1 row)
+
+INSERT INTO test_truncated SELECT * FROM generate_series(11, 13);
+SELECT count(*) FROM test_truncated;
+ count
+---------------------------------------------------------------------
+     3
+(1 row)
+
+-- vacuum full
+CREATE TABLE test_vacuum_full (a int) USING columnar;
+INSERT INTO test_vacuum_full SELECT * FROM generate_series(1, 10);
+SELECT count(*) FROM test_vacuum_full;
+ count
+---------------------------------------------------------------------
+    10
+(1 row)
+
+SELECT relfilenode AS relfilenode_pre_vacuum_full
+FROM pg_class WHERE oid = 'test_vacuum_full'::regclass::oid \gset
+VACUUM FULL test_vacuum_full;
+SELECT relfilenode AS relfilenode_post_vacuum_full
+FROM pg_class WHERE oid = 'test_vacuum_full'::regclass::oid \gset
+SELECT :relfilenode_post_vacuum_full <> :relfilenode_pre_vacuum_full AS relfilenode_changed;
+ relfilenode_changed
+---------------------------------------------------------------------
+ t
+(1 row)
+
+INSERT INTO test_vacuum_full SELECT * FROM generate_series(11, 13);
+SELECT count(*) FROM test_vacuum_full;
+ count
+---------------------------------------------------------------------
+    13
+(1 row)
+
+-- alter column type
+CREATE TABLE test_alter_type (a int) USING columnar;
+INSERT INTO test_alter_type SELECT * FROM generate_series(1, 3);
+SELECT count(*) FROM test_alter_type;
+ count
+---------------------------------------------------------------------
+     3
+(1 row)
+
+SELECT relfilenode AS relfilenode_pre_alter
+FROM pg_class WHERE oid = 'test_alter_type'::regclass::oid \gset
+ALTER TABLE test_alter_type ALTER COLUMN a TYPE text;
+SELECT relfilenode AS relfilenode_post_alter
+FROM pg_class WHERE oid = 'test_alter_type'::regclass::oid \gset
+SELECT :relfilenode_pre_alter <> :relfilenode_post_alter AS relfilenode_changed;
+ relfilenode_changed
+---------------------------------------------------------------------
+ t
+(1 row)
+
+INSERT INTO test_alter_type SELECT * FROM generate_series(17, 19);
```

Comment:
the tests above this have the consistency of inserting the same values before and after the operation. (insert 1-10 before, 11-13 after)

Why are we breaking that consistency here? I kind of liked that the inserted data was the same all the time. If possible I would want that to proceed here. It makes the test easier to read